projects/deja-js/component/popup/model/popup-config.model.ts

```diff
@@ -156,7 +156,7 @@ export class DejaPopupConfig extends MatDialogConfig {
 }
 
 export type DialogToolbarType = 'base' | 'window';
-export type DialogToolbarColor = null | 'primary' | 'accent' | 'warn';
+export type DialogToolbarColor = null | 'primary' | 'accent' | 'warn' | 'orange';
```

Comment:
Ok, you'll make a breaking but it's better than this. 
colors are : primary, accent, warn (orange), danger (red).

Check if there is a usecase inside this and the internal lib'. If you find one, update it.

You can check StatusComponent inside internal lib'.

Reply:
Fait également.